crates/ra_hir/src/nameres.rs

```diff
@@ -136,6 +136,7 @@ pub(crate) struct ModuleData {
 pub struct ModuleScope {
     items: FxHashMap<Name, Resolution>,
     macros: FxHashMap<Name, MacroDef>,
+    textual_macros: FxHashMap<Name, MacroDef>,
```

Comment:
Let's name this `legacy_macros`, and also let's write a short comment explaining the difference between the two macro maps here